static/monorail-2.0.yaml

```diff
@@ -4680,6 +4680,39 @@ paths:
       - Write
       - workflowsModify
     x-swagger-router-controller: workflows
+  /notification/alerts:
```

Comment:
@brianparry @tannoa2 : I didn't realize you were planing to remove the southbound API, but as I know the splitting of southbound API and northbound API is an important part for our forthcoming micro services design. The southbound API is only internal API, it should not be restricted by semantic version, and we may even has some anti-RESTful design if the node ability is limited. If both are merged, if southbound API is still visible to user, how could we apply the difference policy for southbound and northbound? Maybe you already have the solution to handle it within same swagger config file, could you please tell me some detail?